admin_tests/users/test_views.py

```diff
@@ -585,4 +586,4 @@ def test_merge_user(self, mock_merge_user):
         assert valid_form.is_valid()
 
         view.form_valid(valid_form)
-        assert mock_merge_user.called_with()
+        mock_merge_user.assert_called_with()
```

Comment:
For this test, I didn't see why the original tests was wrong previous. We probably need to have a list of tests with same/similar fixes and go through each one by one.

Reply:
Previous tests were wrong because mocks do not have a method called `called_with`.
I'll provide image with illustrates the issue the best

![image](https://github.com/CenterForOpenScience/osf.io/assets/158075011/25e91f1c-0e43-4d09-9f10-82354865937c)

Reply:
ah, I see thanks for the explanation, I didn't get this point during our meeting